src/framework/preferences/css/stylus/PrefsEditor.styl

```diff
@@ -108,21 +108,54 @@
             display:block;
             top:0;
             left:0;
-            height: 1.5em;
-            width: 1.5em;
             margin-top: -0.4em;
             margin-left: -0.4em;
-            background-color: #fff;
-            border: 1px solid #b3b3b3;
-            border-radius: 2em;
-            box-shadow: 4px 2px 3px rgba(0,0,0,0.3);
-            background-image: linear-gradient(right top, rgb(255,255,255) 46%, rgb(233,234,234) 73%);
+            slider-thumb-common()
 
             &:active {
                 outline: none;
             }
         }
     }
+
+    // Native slider appearance
+
+    input[type=range].fl-slider {
+        -webkit-appearance: none; /* Hides the slider so that custom slider can be made */
+    }
+
+    // Native slider thumb appearance
+
+    input[type=range].fl-slider::-webkit-slider-thumb {
+        -webkit-appearance: none;
+        slider-thumb-common()
+    }
+
+    input[type=range].fl-slider::-moz-range-thumb {
+        slider-thumb-common()
+    }
+
+    .fl-prefsEditor input[type=range].fl-slider::-ms-thumb {
+        slider-thumb-common()
+    }
```

Comment:
That causes Stylus to produce the following: 

```
.fl-prefsEditor input[type=range].fl-slider::-webkit-slider-thumb,
.fl-prefsEditor input[type=range].fl-slider::-moz-range-thumb,
.fl-prefsEditor input[type=range].fl-slider::-mms-thumb {
  /* ...css directives /*
}
```

It's valid CSS, but some browsers will throw out the entire directive because of a vendor-specific pseudoclass they don't recognize. The issue described at http://stackoverflow.com/questions/16982449/why-isnt-it-possible-to-combine-vendor-specific-pseudo-elements-classes-into-on, basically.